db/query.go

```diff
@@ -388,6 +388,9 @@ func (context *DatabaseContext) buildChannelsQuery(channelName string, startSeq
 	channelQueryStatement := replaceActiveOnlyFilter(channelQuery.statement, activeOnly)
 	channelQueryStatement = replaceSyncTokensQuery(channelQueryStatement, context.UseXattrs())
 	channelQueryStatement = replaceIndexTokensQuery(channelQueryStatement, index, context.UseXattrs())
+	if ordering != "" {
+		channelQueryStatement = fmt.Sprintf("%s ORDER BY %s", channelQueryStatement, ordering)
```

Comment:
Okay, yeah. I missed the non "*" case when adding that. I don't have a particular scenario but I was under the impression that even if you use an index ordering isn't guaranteed unless you specify `ORDER BY` although it would obviously make sense to be ordered by the index. Would it be worth adding a static `ORDER BY` to guarantee this?

Reply:
Added ORDER BY to QueryStarChannel